blobstore/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-interface-blobstore"
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
Leaving this comment to start, is this a breaking change since bumping a patch version when an actor depends on `wasmbus_rpc 0.9.X` will cause the actor to fail to compile?

Even if there are no breaking changes in wasmbus-rpc from 0.9-0.10, cargo complains that they are mismatched versions if I remember correctly

Reply:
a minor version bump doesn't hurt, so updated them all.